packages/localize/examples/transform/package.json

```diff
@@ -20,6 +20,6 @@
     "rollup": "^2.41.1",
     "rollup-plugin-summary": "^1.3.0",
     "rollup-plugin-terser": "^7.0.2",
-    "typescript": "^4.3.5"
+    "typescript": "~4.3.5"
```

Comment:
Is this what we want to do here long term?

Reply:
Nope, but the typescript rollup plugin is totally broken with TypeScript 4.4, see https://github.com/rollup/plugins/issues/983

Reply:
I see. Too bad there's no comments allowed in json files :/

Reply:
We do want to keep the typescript constraint as a `~x.y.z` though, so this is great. We should have had that before, but relied on the package lock...